### PR TITLE
Update documentation to discourage virtualenv usage

### LINKED ISSUE #6783: Update "getting stated" to discourage virtualenvs/venvs
Update `docs/development/getting-started.rst` (the source for https://warehouse.readthedocs.io/development/getting-started/ ) to say, in the "Configure the development environment" section, something like:

"In case you are used to using a `venv`/virtualenv or virtual environment for Python development: For Warehouse development, don't use one. Our `Makefile` scripts and Docker container development flow create and remove virtualenvs as needed while you are building and testing your work locally."

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

### PR DESCRIPTION
**Pull request by raghav-ys**
Fixes #6783.

### CODE REVIEW

**Comment by brainwane** [comment]
@raghav-ys Please fix the whitespace issue! You can use https://warehouse.readthedocs.io/development/getting-started/#running-tests-and-linters to run the tests locally as you improve your patch.

Just so you know, to update this pull request, you don't need to close it and open a new one. You can [update your pull request](https://zulip.readthedocs.io/en/latest/git/pull-requests.html#update-a-pull-request) and we recommend you do that (instead of closing the old PR and opening a new one) because then it's easier to see the history of what you've done, to to check that you've responded to previous reviewers' comments, and to preserve links we've mentioned (in comments) with other issues and PRs.

**Comment by raghav-ys** [comment]
@brainwane whitespace issue is fixed. Appreciate your guidance.

**Comment by brainwane** [comment]
@alanbato or others, could you help get this PR reviewed and finished? I am busy for several weeks.

File: docs/development/getting-started.rst [hunk 1]
```diff
@@ -58,6 +58,10 @@ you stay up-to-date with our repository:
 Configure the development environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+   Do not use venv/virtualenv for configuring the development environment.
+   Docker creates and remove venv/virtualenv as needed.
```
Comment (ammarnajjar) [inline docs/development/getting-started.rst]: hello @raghav-ys, thanks for taking the time to improve the docs!

Personally, I would actually prefer the wording of @brainwane mentioned in the original issue #6783, for I find it more developer friendly. Reading a direct "Do not" made me stall a bit 😆
More importantly, the suggested text in the original issue is more accurate, for Docker does not create/remove stuff on its own.

Keep up contributing!

**Pull request closed without merging**